tests/unit/django/test_charm.py

```diff
@@ -80,3 +85,36 @@ def test_django_config(harness: Harness, config: dict, env: dict) -> None:
         "after": ["statsd-exporter"],
         "user": "_daemon_",
     }
+
+
+def test_django_create_super_user(harness: Harness) -> None:
+    """
+    arrange: Start the Django charm. Mock the Django command (pebble exec) to create a superuser.
+    act: Run action create superuser.
+    assert: The action is called with the right argumente, returning a password for the user.
```

Comment:
```suggestion
    assert: The action is called with the right argument, returning a password for the user.
```

Reply:
Thanks! I will fix it in the new PR (that is over this branch)